index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+    <title>David Porter</title>
+  </head>
+  <body>
+<section class="pagehead">
+
```

Comment:
This is more of a preference, but you will see a lot of html code without extra line spaces. Might be a good habit to start for a suggestion.